src/calculateRentalCost.js

```diff
@@ -3,8 +3,24 @@
  *
  * @return {number}
  */
+const RENT_PER_DAY = 40;
+const MIDDLE_TERM = 3;
+const MIDDLE_TERM_DISCOUNT = 20;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  const RENT = RENT_PER_DAY * days;
```

Comment:
```suggestion
  const rent = RENT_PER_DAY * days;
```

We use UPPER_CASE only for static values